### ISSUE TITLE
change imports when move folder

### ISSUE THREAD

**Issue by radziksh** [asvetliakov/move-imports#6]
When I move the whole folder  in other place into explorer, then dialog box "Update n references" is not displayed. Is this a bug or this functionality is not implemented? if not implemented - is it possible to implement it?
VSCode 1.14.0
move-imports: 0.0.8

**Comment by asvetliakov** [comment]
It must work. Check both that your original directory and final destination path are not excluded.

**Comment by cspotcode** [comment]
I'm experiencing the same issue.  When I drag-and-drop a file in the sidebar, `move-imports` notifies me to update imports.  However, when I drag-and-drop a directory in the sidebar, nothing happens.

I'm using the default move-imports configuration, installed today, running the latest VSCode.  It's a TypeScript project; all files are `.ts` extension.

VSCode version: 1.17.1
move-imports version: 0.0.9
OS: Windows 10

**Comment by asvetliakov** [comment]
can you share your .gitignore and sample directory tree structure?

**Comment by cspotcode** [comment]
Here's a minimal reproduction:

![image](https://user-images.githubusercontent.com/376504/31693115-e75cc6ca-b36a-11e7-85ea-216ed0413697.png)

There is not a `.gitignore`, `node_modules`, or `package.json`.  The only files are `foo.ts` and `bar.ts`.  Dragging `drag-me-to-dirB` onto `dirB`, moving it from `./dirA/drag-me-to-dirB` to `./dirB/drag-me-to-dirB`, does not rewrite the import statement in `foo.ts`.

**Comment by asvetliakov** [comment]
![screen shot 2017-10-18 at 00 48 24](https://user-images.githubusercontent.com/8881674/31693307-1e158390-b39e-11e7-9f55-10e99eb41092.png)
The result must look like this? At least it works on mac, will try on windows tomorrow

**Comment by cspotcode** [comment]
Yes, that's what I expect to happen, but instead nothing happens.
